### PR TITLE
feat: ui-for-change-pin-options

### PR DESCRIPTION
**Pull request by HaseebUllahAbbasi**

### CODE REVIEW

**Review by sadiabbasi — APPROVED** [review]

File: src/navigation/Pin.tsx
Comment (Rebal67) [inline src/navigation/Pin.tsx:1]: cant all of them be at the settings navigator ? as the navigation type doesnt change?

**Review by Rebal67 — COMMENTED** [review]
as stated in the design in Figma there is space between the header and the navigators
also the color of the active buttons are bolder than the disabled buttons can you add that

File: src/navigation/Pin.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import PinSettingsScreen from '../screens/PinSettingsScreen';
+
+export type RouteStackParamList = {
+    AddPin: undefined;
+    Splash: undefined;
+    Settings: undefined;
+    PinSettings: undefined;
+    ChangePin: undefined;
+    RemovePin: undefined;
+};
```
[the remark this answers is not in the document]
Comment (HaseebUllahAbbasi) [reply src/navigation/Pin.tsx:19]: removed

File: src/navigation/Pin.tsx
Comment (HaseebUllahAbbasi) [reply src/navigation/Pin.tsx:1]: the navigation is nested here so thats why i used other here,

**Pull request merged**